src/CloudFlare/IpRewrite.php

```diff
@@ -50,14 +50,14 @@ public function isCloudFlare()
             return false;
         }
 
-        return true;
+        return $this->isCloudFlareIP();
```

Comment:
README file should also be edited.

```
isCloudFlare(); returns true if the CF_CONNECTING_IP header is present in the request.
```

is no longer true since it checks the valid cf ip's